apps/create/widget/layouts/Container.jsx

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
This is the refactor/structure that I'm really starting to like for Templates.

A file that is very structural and has a clear syntax of what is being displayed, -- complex types get passed in as components in props. Now it should be really easy to swap templates, as long as it gets passed the same Components.

I think it also helps centralize all of the business logic in the parent, instead of functions needing to be passed through multiple layers

Reply:
I agree with you I like this structure too for easy swappable parts. How can we integrate this concept in the user projects?